examples/ovsbridge_bond_create.yml

```diff
@@ -0,0 +1,18 @@
+---
+interfaces:
+  - name: ovs-br0
+    type: ovs-bridge
+    state: up
+    bridge:
+      options:
+        stp: false
+      port:
+        - name: ovs-bond1
+          type: bond
+          link-aggregation:
+            mode: balance-slb
+            slaves:
+              - name: eth1
+                type: system
```

Comment:
Do we need type here ?

Reply:
We discussed it with Edy aboved. I want to keep it identical to regular ports. Then we can see if we can get a rid of "type" from both slaves and ports.

Reply:
ack